src/sass/components/_timeline.scss

```diff
@@ -0,0 +1,215 @@
+// Copyright (C) 2018 The Trustees of Indiana University
+// SPDX-License-Identifier: BSD-3-Clause
+
+.#{$prefix}-timeline {
+  position: relative;
+  padding: $lg 0;
+
+  &::before {
+    content: '';
+    margin-left: $xxs;
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: $xxs;
+    background: $color-black--200;
+  }
+
+  &__item::after {
+    margin-left: 0;
+    border: solid transparent;
+    content: " ";
+    height: 0;
+    width: 0;
+    position: absolute;
+    pointer-events: none;
+    border-right-color: $color-black--100;
+    border-width: $sm;
+    top: .4rem;
+  }
+
+  &__item {
+    position: relative;
+    margin: $lg 0;
+  }
+
+  /**
+   *  The marker for the timeline visually only looks good at 12px (.75rem). This results in several magic numbers related to the placement of other items such as the &__item::after (above) and date labels (where applicable). This also results in cascading magic numbers for placement of elements such as &__content for variant timelines.
+   */
+
+  &__marker {
+    width: $sm*.75;
+    height: $sm*.75;
+    position: absolute;
+    color: $color-white;
+    background: $color-midnight;
+    border-radius: 50%;
+    box-shadow: 0 0 0 $xxs $color-black--100;
+    margin-top: $sm;
+  }
+
+  &__content {
+    padding: $md;
+    margin-left: $lg;
+    background-color: $color-white;
+    border-radius: $xxs;
+    box-shadow: $shadow-base;
+    border: none;
+  }
+
+  & h2 {
+    font-size: $ts-26;
+    font-weight: $font-weight-bold;
+  }
+
+  &__date {
+    font-size: $ts-14;
+    color: $color-black--700;
+  }
+}
+
+.#{$prefix}-timeline--right {
+  &::before {
+    left: unset;
+    right: $xxs;
+  }
+
+  .#{$prefix}-timeline__item::after {
+    margin-left: 0;
+    right: 0;
+    border-left-color: $color-black--100;
+    border-right-color: transparent;
+  }
+
+  .#{$prefix}-timeline__marker {
+    margin-left: 0;
+    right: 0;
+  }
+
+  .#{$prefix}-timeline__content {
+    margin-left: 0;
+    margin-right: $lg;
+  }
+}
+
+@include mq($breakpoint-md) {
+  .#{$prefix}-timeline--center {
+    &::before {
+      left: 50%;
+      margin-left: 0;
+    }
+
+    & > div {
```

Comment:
This feels a bit off. Is there a specific reason not to set `--timeline-center` to `display: flex` and  then also set `__item` to `display: flex`? It is possible for an element to be both a flex container and a flex child. This would need some testing to confirm, but similar to my previous comment on super generic selectors, could we tie this styling to an explicit class name that is already being used in the component?

Reply:
The reason I did it this way was to ensure that each item is in its own row, rather than stacking up together:

<img width="1122" alt="Screen Shot 2019-07-24 at 3 44 22 PM" src="https://user-images.githubusercontent.com/15944675/61823558-02c78380-ae2a-11e9-9e27-c878dda6385d.png">